website/content/docs/getting-started.md

```diff
@@ -156,9 +156,9 @@ $ heron-tracker
 ... Using config file: $HOME/.herontools/conf/heron_tracker.yaml
 ```
 
-You can reach Heron Tracker in your browser at [http://localhost:8888](http://localhost:8888)
-and see something like the following upon successful submission of the topology: 
-![alt tag](http://twitter.github.io/heron/img/heron-tracker.png)
+You can reach Heron Tracker in your browser at ``http://localhost:8888``
```

Comment:
Adding API docs - along side 'Get Started' is not a good idea. Instead, it might be better if you could add it at the top next to 'Docs'. Instead of calling it Docs, call it just an 'API'